src/app/api/monitoring/migration-success/route.ts

```
@@ -0,0 +1,207 @@
+    .gte('created_at', timeRange.start.toISOString());
+  
+  const v1AvgSimilarity = v1Metrics && v1Metrics.length > 0
+    ? v1Metrics.reduce((sum, m) => sum + m.avg_similarity, 0) / v1Metrics.length
+    : 0;
+  
+  const v2AvgSimilarity = v2Metrics && v2Metrics.length > 0
+    ? v2Metrics.reduce((sum, m) => sum + m.avg_similarity, 0) / v2Metrics.length
+    : 0;
+  
+  // ゼロ除算防止
+  const safeV1Avg = isFinite(v1AvgSimilarity) ? v1AvgSimilarity : 0;
```

Biome (1.9.4)

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)